content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/_cq_design_dialog/.content.xml

```diff
@@ -74,11 +77,30 @@
                                         <datasource
                                             jcr:primaryType="nt:unstructured"
                                             sling:resourceType="core/wcm/components/embed/v1/datasources/embeddables"/>
+                                        <granite:data
+                                            jcr:primaryType="nt:unstructured"
+                                            cmp-embed-dialog-edit-showhidetarget="\[data-cmp-embed-dialog-edit-youtube\]"/>
                                     </allowedEmbeddables>
                                 </items>
                             </column>
                         </items>
                     </properties>
+                    <youTubeParametersTab
```

Comment:
I think it would be nice to dynamically append such tabs based on the allowed embeddables.

Reply:
Indeed that would be nice, but the problem is you cannot combine literal items with dynamic data sources (https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/granite-ui/api/jcr_root/libs/granite/ui/docs/server/datasource.html). In case we want to have the tabs dynamic also the first tab of the embeddable would need to be returned by a Data Source servlet, WDYT?

Reply:
Done in https://github.com/adobe/aem-core-wcm-components/pull/1301/commits/525037cc7e2688e8c475724980e8f1d4196d7c90